SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditor.h

```diff
@@ -10,12 +10,12 @@ DECLARE_LOG_CATEGORY_EXTERN(LogSpatialGDKEditor, Log, All);
 
 DECLARE_DELEGATE_OneParam(FSpatialGDKEditorErrorHandler, FString);
 
+class FSpatialGDKPackageAssembly;
+
 class SPATIALGDKEDITOR_API FSpatialGDKEditor
 {
 public:
-	FSpatialGDKEditor() : bSchemaGeneratorRunning(false)
-	{
-	}
+	FSpatialGDKEditor();
```

Comment:
why did that get changed? / does this mean that we now run schema generation when starting the editor?

Reply:
It is moved into the cpp file because we have additional instantiation happening in initialization and we don't want to include the full definition of the PackageAssembly class in the header of the editor.